ui/helpers/utils/util.test.js

```diff
@@ -340,4 +340,101 @@ describe('util', () => {
       expect(util.toHumanReadableTime(t, 7200000)).toStrictEqual('2 hrs');
     });
   });
+  describe('sanitizeMessage', () => {
+    const message = {
```

Comment:
Nit: Better to move these fixtures to a function, and call this function in each test to get a fresh copy, just in case one of the tests mutates the object.

If these get mutated by one of the tests, it could invalidate the test result, and it might also be very difficult to track down the issue. It could also lead to dependencies between tests.